src/main/webapp/secure/aad.jsp

```diff
@@ -6,19 +6,36 @@
 </head>
 <body>
 
-	<h1>Directory - Users List</h1>
+	<h3>List of users registered in the tenant - ${tenant}</h3>
 	<p>${users}</p>
+	<br>
+	<h3>User who authorised access to graph API</h3>
```

Comment:
Or maybe I misunderstand. Is this the contents of the id_token claims? If so, this should say "Current user" or something similar.